QUICKSTART-ECS.md

```diff
@@ -0,0 +1,191 @@
+# Using a Bottlerocket AMI with Amazon ECS
+
+> The [ECS variant](./aws-ecs.md) of Bottlerocket is currently in a developer preview phase and we're looking for your
+> [feedback](https://github.com/bottlerocket-os/bottlerocket#contact-us).
+> We'd love for you to try it out!
+
+[Amazon Elastic Container Service (Amazon ECS)](https://ecs.aws) is a highly scalable, fast, container management service that makes it easy to run, stop, and manage containers on a cluster.
+Your containers are defined in a task definition which you use to run individual tasks or as a service.
+
+This quickstart will walk through setting up an Amazon ECS cluster with Bottlerocket container instances (using the EC2 launch type).
+Check out the [Amazon ECS developer guide](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) for more information on the supported launch types, additional details, and a broad overview of Amazon ECS.
+
+## Prerequisites
```

Comment:
Also need aws-cli; copy from the existing quickstart?

Reply:
Done, though I took out the reference to v1 and the request to use a recent one; any CLI newer than January 2015 is recent enough to work with ECS.